com.kms.cura.dal/src/com/kms/cura/dal/database/WorkingHourDatabaseHelper.java

```diff
@@ -0,0 +1,72 @@
+package com.kms.cura.dal.database;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.kms.cura.dal.mapping.Doctor_FacilityColumn;
+import com.kms.cura.entity.DayOfTheWeek;
+import com.kms.cura.entity.Entity;
+import com.kms.cura.entity.OpeningHour;
+
+public class WorkingHourDatabaseHelper extends DatabaseHelper{
+
+	public WorkingHourDatabaseHelper() throws ClassNotFoundException, SQLException {
+		super();
+	}
+
+	@Override
+	protected Entity getEntityFromResultSet(ResultSet resultSet) throws SQLException, ClassNotFoundException {
+		return null;
+	}
+	
+	public OpeningHour getWorkingHourFromResultSet(ResultSet rs) throws SQLException, ClassNotFoundException {
```

Comment:
@hieumai  can not put it into the method getEntityFromResultSet because the OpeningHour does not extend Entity